src/SlackConnector/Models/SlackAttachmentActionStyle.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace SlackConnector.Models
+{
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum SlackAttachmentActionStyle
+    {
+        @default,
```

Comment:
Again, to follow naming standards can you make these values CamelCase and maybe use a converter to lowercase if you need lowercase (or use something like `[EnumMember(Value = "default")]`)